### ISSUE TITLE
How to use this with Meteor React

### ISSUE THREAD

**Issue by sonipriya10** [xcv58/meteor-auth0#14]
I am unable to get the profile data fetched from google after using the package. CurrentUser.services.auth0 is showing the property undefined error.
Can you please provide me the sample with react?
I am in hurry

**Comment by sonipriya10** [comment]
Yes I am getting the data in MongoDB, but unable to get the data from mongodb to react app.
Please provide the soolution asap. as I am in middle of the project and I have to submit it soon. Thank You.

**Comment by sonipriya10** [comment]
Is it possble for you to provide an example with react? That will be very kind of you.

**Comment by xcv58** [comment]
I think you need to publish the filed you need, for example:

https://github.com/xcv58/meteor-auth0/blob/805fdf66ff11eb1f7f73cfefd503d62767008075/auth0-lock/auth0-lock-server.js#L60

**Comment by sonipriya10** [comment]
Can you provide me the sample page as you have made with blaze for using this with react?

**Comment by xcv58** [comment]
If you can't see the data in front end then it's not relating to Blaze or React.

Could you verify whether you can get data?

**Comment by sonipriya10** [comment]
I am not getting data in front end. The data is getting save in MongoDB under users->services->auth0

**Comment by sonipriya10** [comment]
Will try and let You know. Can you provide a sample page with react getting profile data.

**Comment by xcv58** [comment]
As I mentioned before the data and UI rendering is not related.  I don't think a React example will make any difference.

**Comment by sonipriya10** [comment]
I am asking this because when I am using the exact same sample app which you have provided everything is working fine. I am getting data also in frontend. Thatwhy I am asking you to provide me an example with react.

**Comment by sonipriya10** [comment]
Hey please can you help, I am unable to get data using currentUser in frontend?

**Comment by sonipriya10** [comment]
In my file

login(e){
    e.preventDefault();
    const Lock = initLock();

     Lock.show();
   }

it is showing initLock is not a function?

I have imported the package also.

**Comment by xcv58** [comment]
Can you log what's `initLock` and please share how you import the package?

**Comment by sonipriya10** [comment]
initLock is a function created as shown in your sample app.

import React,{Component} from 'react';
import {meteor} from 'meteor/meteor';
import {initLock} from 'meteor/xcv58:auth0-lock';

**Comment by xcv58** [comment]
I'm confused. You said `initLock` is not function but is a function now.
